eta/modules/apply_classifier_to_objects.py

```diff
@@ -174,6 +174,8 @@ class ParametersConfig(Config):
             in pixels, for a bounding box to be classified
         confidence_threshold (eta.core.types.Number): [None] the minimum
             confidence required for a label to be saved
+        record_top_k_probs (eta.core.types.Number): [None] top-k class
```

Comment:
Slight clarification for the description: "top-k class probabilities to record for the predictions. (`record_top_k_probs` is the number `k`)."

Reply:
addressed